budgetportal/models/pages.py

```diff
@@ -0,0 +1,145 @@
+from blocks import SectionBlock
+from budgetportal import nav_bar
+from collections import OrderedDict
+from django.core.exceptions import ValidationError
+from django.db import models
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.core import blocks as wagtail_blocks
+from wagtail.core.fields import RichTextField, StreamField
+from wagtail.core.models import Page
+from wagtail.contrib.routable_page.models import RoutablePage, route
+
+
+from .gov_structure import FinancialYear
+
+
+class WagtailHomePage(RoutablePage):
+    max_count = 1
+
+
+class LearningIndexPage(RoutablePage):
+    max_count = 1
+
+
+class PostIndexPage(RoutablePage):
+    max_count = 1
+    parent_page_types = []
```

Comment:
I don't think we want this any more - this makes it impossible to create the page - perhaps it's ok if you create it using a fixture, but I think max_count will make sure only one can be created in any case (and like this, it can't be re-created without a code change if it's deleted for some reason.

Or did I miss something?

Reply:
hm... yeah, probably.